pyei/plot_utils.py

```diff
@@ -83,7 +125,7 @@ def plot_boxplot(voting_prefs_group1, voting_prefs_group2, group1_name, group2_n
     if ax is None:
         ax = plt.gca()
     samples_df = pd.DataFrame({group1_name: voting_prefs_group1, group2_name: voting_prefs_group2})
-    ax = sns.boxplot(data=samples_df, orient="h", whis=[5, 95], ax=ax)
+    ax = sns.boxplot(data=samples_df, orient="h", whis=[2.5, 97.5], ax=ax)
```

Comment:
This is the first "rider change" where I realized that the whiskers were showing a 90% credible interval, not a 95% credible interval!

Reply:
Nice catch